examples/layers.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
can you remove the `layers.ipynb` and replace with additional script examples if necessary